src/components/cid/Cid.js

```diff
@@ -17,17 +21,21 @@ export function shortCid (value) {
   return `${start}…${end}`
 }
 
-const Cid = ({ value, title, style, ...props }) => {
+const Cid = ({ value, title, style, identicon = true, ...props }) => {
   style = Object.assign({}, {
-    textDecoration: 'none'
+    textDecoration: 'none',
+    marginLeft: identicon ? '5px' : null
   }, style)
   const { start, end } = cidStartAndEnd(value)
   return (
-    <abbr title={title || value} style={style} {...props}>
-      <span>{start}</span>
-      <span className='o-20'>…</span>
-      <span>{end}</span>
-    </abbr>
+    <>
+      { identicon && <Identicon string={value} size={20} palette={identiconPalette} /> }
```

Comment:
minor, but let's capture the ipfs flavour identicon with it's colour pallete in it's own component, and import that here, and set the identicon parameter default to false, as we don't want identicions everywhere this component is currently used.